src/ui/public/vis/response_handlers/basic.js

```diff
@@ -68,25 +73,24 @@ const BasicResponseHandlerProvider = function (Private) {
       return new Promise((resolve) => {
         if (vis.isHierarchical()) {
           // the hierarchical converter is very self-contained (woot!)
+          // todo: it should be updated to be based on tabified data just as other responseConverters
```

Comment:
nit: Could you please create an issue or open a WIP PR for that? so we don't miss this.

Reply:
https://github.com/elastic/kibana/issues/22176